README.md

```diff
@@ -89,6 +89,10 @@ const style = css({
 })
 ```
 
+### Images and other static files
+
+Create a folder called `static` in your project root directory. You can place images and other static assets there. Reference them from JS files with the path to the static directory. For example, from `./components` use `../static/file-name.jpg` or from `./pages` simpy use `/static/file-name.jpg`
```

Comment:
ok, I'm taking care of these.

Just curious - where do you you put components? Not in pages, right?